src/Tribe/Aggregator/Record/Async_Queue.php

```diff
@@ -370,6 +370,10 @@ public function has_errors() {
 		return ! empty( $this->error );
 	}
 
+	public function is_fetching() {
```

Comment:
* :no_entry_sign: Line indented incorrectly; expected 0 tabs, found 1
* :no_entry_sign: Missing doc comment for function is_fetching()

<sub>hash: bb5bfccd4c1c615809c32aec6317ac72</sub>